### DISCUSSION TITLE
VO specific container stetup

### DISCUSSION THREAD

**Discussion by panta-123** [DIRACGrid/DIRAC#6085] [Q&A] [answered]
I am looking through the container setup for developer. At Belle2 we are running v7r1 but soon we will have v7r2.
But over server still runs python2.  I have been looking at https://github.com/DIRACGrid/DIRAC/blob/rel-v7r2/integration_tests.py .
Could you provide some more info on setting up VO DIRAC server using docker for python2 ?
(May be some example from specific VO setup ?

**Comment by fstagni** [comment, marked as answer]
`integration_tests.py` is a tool for setting up a test environment (using docker compose) and for running integration tests. Its use is:
- for developers, and it has been documented in [DIRAC docs 1](https://dirac.readthedocs.io/en/latest/DeveloperGuide/CodeTesting/index.html) , [DIRAC docs 2](https://dirac.readthedocs.io/en/latest/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/stuffThatRun.html) and at the latest workshop ([pres](https://indico.cern.ch/event/1107386/contributions/4827835/attachments/2439551/4179565/2022-05-9_vDUW2-Python2_DIRACOS_and_FAQ.pdf) and [video](https://indico.cern.ch/event/1107386/attachments/2438979/4183008/video1768508409.mp4))
- for running the integration tests in gitlab actions: https://github.com/DIRACGrid/DIRAC/actions/workflows/integration.yml
As such is "only" a test tool. 
The integration tests do install a local server (py2 or py3) and client using a fake CA.
